src/main/java/edu/harvard/iq/dataverse/DatasetServiceBean.java

```diff
@@ -699,7 +699,7 @@ public void exportAllDatasets(boolean forceReExport) {
         Integer countError = 0;
         String logTimestamp = logFormatter.format(new Date());
         Logger exportLogger = Logger.getLogger("edu.harvard.iq.dataverse.harvest.client.DatasetServiceBean." + "ExportAll" + logTimestamp);
-        String logFileName = "../logs" + File.separator + "export_" + logTimestamp + ".log";
+        String logFileName = System.getProperty("com.sun.aas.instanceRoot") + File.separator + "logs" + File.separator + "export_" + logTimestamp + ".log";
```

Comment:
I can vouch for the need for this change. Without it, I couldn't get some code I was playing with to execute. I was exporting some JSON for troubleshooting: https://github.com/pdurbin/dataverse/commit/8adb28d67522936b7e3c6f0aa98e8ab720a1982a

Reply:
Sounds like within a container, the application has a different working directory than in a classic deployment. Which means `../logs` may point to some place that does not exist / the app user has no permissions to write to.